<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>QMainWindow</class>
 <widget class="QMainWindow" name="QMainWindow">
  <property name="enabled">
   <bool>true</bool>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1200</width>
    <height>800</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>1200</width>
    <height>800</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>1200</width>
    <height>800</height>
   </size>
  </property>
  <property name="font">
   <font>
    <family>Montserrat Medium</family>
   </font>
  </property>
  <property name="windowTitle">
   <string/>
  </property>
  <property name="styleSheet">
   <string notr="true">#LoginDiv {
	background-color: #111114;
	border-radius: 10px;
}

#ButtonClose {
	background-color: rgba(255, 255, 255, 0)
}

#NavBar {
	background-color: #111114;
}

#NavBar_Title {
	color: #595963;
}

#PasswordLabel {
	color: #fff;
	font-size: 16px;
}

#UsernameLabel {
	color: #fff;
	font-size: 16px;
}

#LoginButton {
	border-radius: 15px;
	background-color: #18181c;
    color: #fff;
	font-size: 16px;
}

#LoginButton:hover {
	background-color: #1f1f23;
    color: #fff;
}

QLineEdit {
	border-radius: 15px;
	background-color: #18181c;
    color: #fff;
	font-size: 16px;
	padding: 5px 10px;
	white-space: none;
}</string>
  </property>
  <property name="animated">
   <bool>true</bool>
  </property>
  <widget class="QWidget" name="Main">
   <widget class="QPushButton" name="LoginButton">
    <property name="geometry">
     <rect>
      <x>500</x>
      <y>500</y>
      <width>201</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Montserrat Light</family>
      <pointsize>-1</pointsize>
      <weight>75</weight>
      <bold>true</bold>
      <kerning>true</kerning>
     </font>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="text">
     <string>LOGIN</string>
    </property>
    <property name="default">
     <bool>false</bool>
    </property>
    <property name="flat">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QLineEdit" name="Username">
    <property name="geometry">
     <rect>
      <x>480</x>
      <y>330</y>
      <width>241</width>
      <height>41</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="Password">
    <property name="geometry">
     <rect>
      <x>480</x>
      <y>410</y>
      <width>241</width>
      <height>41</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="Logo">
    <property name="geometry">
     <rect>
      <x>470</x>
      <y>150</y>
      <width>281</width>
      <height>131</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap>../img/logo.png</pixmap>
    </property>
   </widget>
   <widget class="QLabel" name="Background">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>1200</width>
      <height>800</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>8</pointsize>
     </font>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap>../img/login-background.png</pixmap>
    </property>
   </widget>
   <widget class="QLabel" name="LoginDiv">
    <property name="geometry">
     <rect>
      <x>450</x>
      <y>275</y>
      <width>300</width>
      <height>301</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="UsernameLabel">
    <property name="geometry">
     <rect>
      <x>480</x>
      <y>380</y>
      <width>241</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Montserrat Medium</family>
      <pointsize>-1</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Password:</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="PasswordLabel">
    <property name="geometry">
     <rect>
      <x>480</x>
      <y>300</y>
      <width>241</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Montserrat Medium</family>
      <pointsize>-1</pointsize>
      <weight>50</weight>
      <italic>false</italic>
      <bold>false</bold>
      <underline>false</underline>
      <strikeout>false</strikeout>
      <kerning>true</kerning>
     </font>
    </property>
    <property name="text">
     <string>Username:</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="NavBar_Title">
    <property name="geometry">
     <rect>
      <x>4</x>
      <y>4</y>
      <width>61</width>
      <height>18</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Montserrat SemiBold</family>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="cursor">
     <cursorShape>OpenHandCursor</cursorShape>
    </property>
    <property name="mouseTracking">
     <bool>true</bool>
    </property>
    <property name="text">
     <string> ScapeX</string>
    </property>
   </widget>
   <widget class="QLabel" name="NavBar">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>1201</width>
      <height>25</height>
     </rect>
    </property>
    <property name="cursor">
     <cursorShape>OpenHandCursor</cursorShape>
    </property>
    <property name="mouseTracking">
     <bool>true</bool>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="ButtonClose">
    <property name="geometry">
     <rect>
      <x>1175</x>
      <y>0</y>
      <width>25</width>
      <height>25</height>
     </rect>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="icon">
     <iconset>
      <normaloff>../img/close-icon.png</normaloff>../img/close-icon.png</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>25</width>
      <height>25</height>
     </size>
    </property>
   </widget>
   <zorder>Background</zorder>
   <zorder>NavBar</zorder>
   <zorder>LoginDiv</zorder>
   <zorder>LoginButton</zorder>
   <zorder>Username</zorder>
   <zorder>Password</zorder>
   <zorder>Logo</zorder>
   <zorder>UsernameLabel</zorder>
   <zorder>PasswordLabel</zorder>
   <zorder>NavBar_Title</zorder>
   <zorder>ButtonClose</zorder>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
